packages/ui5-config/src/ui5config.ts

```diff
@@ -91,7 +91,7 @@ export class UI5Config {
             libraryObjs.push({ name: library });
         }
         // Add theme lib (dark theme versions are provided by base theme lib)
-        libraryObjs.push({ name: `themelib_${ui5Theme.replace(/_dark$/, '')}` });
+        libraryObjs.push({ name: `themelib_${ui5Theme.replace(/_dark|_hcw|_hcb/, '')}` });
```

Comment:
Are you sure about dropping the `$`. This changes the regex match to match anywhere in the string not just the end. I think we should retain the precision.

Reply:
Thanks @IainSAP, i have added it back